lib/content/utilities.ex

```diff
@@ -110,4 +110,17 @@ defmodule Content.Utilities do
 
   def render_datetime_as_time(time),
     do: Calendar.strftime(time, "%I:%M") |> String.replace_leading("0", "")
+
+  def crowding_description_var(crowding_description) do
+    case crowding_description do
+      {:front, _} -> "870"
+      {:back, _} -> "871"
+      {:middle, _} -> "872"
+      {:front_and_back, _} -> "873"
+      {:train_level, :not_crowded} -> "874"
+      {:train_level, :some_crowding} -> "875"
+      {:train_level, :crowded} -> "876"
+      _ -> nil
```

Comment:
We should probably make this a "space" var so it's valid, since we don't have any special error checking for this case.